server/server.js

```diff
@@ -96,8 +96,11 @@ app.get('/', (req, res) => {
   res.render('index');
 });
 
-app.get('/unsupported', (req, res) => {
-  res.render('unsupported');
+app.get('/unsupported/:reason', (req, res) => {
+  const outdated = req.params.reason === 'outdated'? true : false;
```

Comment:
I think the `? true : false` sorcery is unnecessary here since `req.params.reason === 'outdated'` should be Boolean-enough(tm).